### PR TITLE
Remove invitation by e-mail at members page

### PR DESCRIPTION
**Pull request by brnsolikyl**
Solves (partially) #2080 

## What does this PR do?

Removes invitation by e-mail and replaces it with invitation by copying link to `groupPreview` page.

To do:
- [x] add new text to locale
- [x] figure out how to get and display the complete URL for the groupPreview
- [x] add some visual feedback to show the link has been copied to clipboard
- [x] add close button to dialog

## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [ ] no unrelated changes
- [x] asked someone for a code review
- [x] joined [chat.foodsaving.world/channel/karrot-dev](https://chat.foodsaving.world/channel/karrot-dev)
- [x] added an entry to CHANGELOG.md (description, pull request link, username(s))
- [x] tried out on a mobile device (does everything work as expected on a smaller screen?)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Please document your changes in this format:
 ```
 
 ## [Unreleased]
+### Changed
+- remove invitation by e-mail at members page and substitute it for a group link to be copied @brnsolikyl [#2349]
```
Comment (larzon83) [inline CHANGELOG.md:25]: I would just write it as the PR title `Remove invitation by e-mail at members page` - more info is also in the PR

File: src/locales/locale-en.json [hunk 2]
```diff
@@ -382,6 +382,8 @@
     "INVITATIONS": "Invitations",
     "INVITATION_ACCEPT_ERROR": "Ooops, could not accept the invitation!",
     "INVITATION_ACCEPT_SUCCESS": "Welcome! You successfully accepted the invitation.",
+    "INVITATION_COPY_LINK": "Copy the link and send it to the person you want to invite.",
```
Comment (larzon83) [inline src/locales/locale-en.json:385]: I would go with another level like so:
```json
"INVITATION_DIALOG": {
  "MESSAGE": "Copy the link and send it to the person you want to invite.",
  "TITLE": "Invitation link"
}
```

File: src/users/pages/Members.vue [hunk 3]
```diff
@@ -14,20 +14,63 @@
           :title="$t(sorting === 'joinDate' ? 'GROUP.SORT_NAME' : 'GROUP.SORT_JOINDATE')"
           @click="toggleSorting"
         />
-        <RouterLink
-          v-if="isEditor"
```
Comment (larzon83) [inline src/users/pages/Members.vue:18]: Is the removal of checking for `isEditor` intentional? I think it makes sense. Just double checking ;)

File: src/users/pages/Members.vue [hunk 4]
```diff
@@ -59,11 +106,15 @@ export default {
...
     QCard,
+    QCardSection,
+    QDialog,
+    QField,
     QBtn,
   },
   data () {
     return {
       sorting: 'joinDate',
+      inviteLink: false,
```
Comment (larzon83) [inline src/users/pages/Members.vue:120]: `inviteDialog` would be a better fit.

File: src/users/pages/Members.vue [hunk 5]
```diff
@@ -14,20 +14,63 @@
...
+        <QBtn
+          small
+          round
+          color="secondary"
+          icon="fas fa-user-plus"
+          :title="$t('GROUP.INVITE_TITLE')"
+          @click="inviteLink = true"
+        />
+
+        <QDialog v-model="inviteLink">
```
Comment (larzon83) [inline src/users/pages/Members.vue:26]: maybe switch to using `CustomDialog` component as this has predefined layout and header styling included? See eg here: https://github.com/yunity/karrot-frontend/pull/2368
I could help if you want, but fine with the current solution, too.

**Review by larzon83 — CHANGES_REQUESTED** [review]
Looks good. Just some minors :)

File: src/users/pages/Members.vue (hunk 3 above)
Comment (brnsolikyl) [reply src/users/pages/Members.vue:18]: Yes. Inviting people by groupPreview link should be available to anyone in the group. :)

File: src/users/pages/Members.vue (hunk 5 above)
Comment (brnsolikyl) [reply src/users/pages/Members.vue]: Yep! Now using `CustomDialog`. Everything looks fine here, but you might want to double-check

**Comment by brnsolikyl** [comment]
Thx for the review, @larzon83 ! I changed everything accordingly.

**Review by larzon83 — APPROVED** [review]

**Pull request merged**